tensorflow_upgrade.md

```diff
@@ -0,0 +1,24 @@
+## Tensorflow upgrade
+
+Current Version: 1.4.1
+
+Versions Tested:
+ - **1.5.0**
+ - **1.6.0**
+ - **1.7.0**
+ - **1.8.0** 
+    - **Remarks:** Upgrading to version 1.5.0-1.8.0 breaks no functionality and produces no error when running with Travis CI. Importing data models also do not seem to produce any problems.
+ - **1.9.0** & **1.10.1** 
+    - **Errors encountered:** An `ImportError: dlopen: cannot load any more object with static TLS_` was thrown when trying to run the project.
+    - **Cause of error:** This error seems to be thrown due to a wrong order of import statements. This error seems to be a bug as none of the breaking changes from v1.8.0 to 1.9.0 ([source](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md#breaking-changes-2)) apply. On upgrading to 1.12.0, this issue no longer persisted. 
```

Comment:
Can you rephrase this `This error seems to be thrown due to a wrong order of import statements. This error seems to be a bug` as `This error seems to be thrown due to a wrong order of import statements, which seems to be a bug..`